scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -1240,16 +1249,16 @@ def test_align_vectors_improper_rotation():
 
 
 def test_align_vectors_scaled_weights():
-    rng = np.random.RandomState(0)
-    c = Rotation.random(random_state=rng)
-    b = rng.normal(size=(5, 3))
-    a = c.apply(b)
+    n = 10
+    a = Rotation.random(n, random_state=0)
+    b = Rotation.random(n, random_state=1)
+    scale = 2
 
-    est1, rmsd1, cov1 = Rotation.align_vectors(a, b, np.ones(5), True)
-    est2, rmsd2, cov2 = Rotation.align_vectors(a, b, 2 * np.ones(5), True)
+    est1, rssd1, cov1 = Rotation.align_vectors(a, b, np.ones(n), True)
+    est2, rssd2, cov2 = Rotation.align_vectors(a, b, scale * np.ones(n), True)
 
     assert_allclose(est1.as_matrix(), est2.as_matrix())
-    assert_allclose(np.sqrt(2) * rmsd1, rmsd2)
```

Comment:
Because the original test formed `a` via an exact rotation of `b`, rmsd was always 0, so this was an ineffective assert. Updated to better test this.